features/step_definitions/settings_steps.rb

```diff
@@ -20,5 +20,14 @@
 end
 
 Then "{provider} should have strong passwords {enabled}" do |provider, enabled|
-  assert provider.settings.strong_passwords_enabled == enabled
+  assert provider.settings.reload.strong_passwords_enabled == enabled
+end
+
+Given "{provider} has {int} account plan(s)" do |provider, count|
+  if count == 1
+    first_plan_id = provider.account_plans.first.id
+    provider.account_plans.where.not(id: first_plan_id).destroy_all
+  else
+    FactoryBot.create_list(:account_plan, count, provider: provider)
```

Comment:
I'm sorry for nitpicking :grimacing: 
If we know that provider has one or more accounts originally, I guess we need to know how many and then create the remaining count only.

If step is `one|multiple` instead of exact int, this is fine for me.

Reply:
I addressed this in https://github.com/3scale/porta/pull/3666/commits/8dd597fc0645665332b2e562481992c229800b48